src/main/java/br/fatec/bd4/repository/RegistroRepository.java

```diff
@@ -2,22 +2,33 @@
 
 import br.fatec.bd4.entity.Registro;
 
+import br.fatec.bd4.web.dto.MaxMinDTO;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.Optional;
+
 @Repository
 public interface RegistroRepository extends JpaRepository<Registro, Long> {
 
+
+    @Query(
```

Comment:
It's missing the params dataInicio and dataFim. Could you please check it?

Reply:
I'll fix that.